Doc/library/enum.rst

```diff
@@ -622,3 +635,29 @@ Utilites and Decorators
       ...
       ValueError: duplicate values found in <enum 'Mistake'>: FOUR -> THREE
 
+.. decorator:: simple_enum
+
+   A :keyword:`class` decorator that converts a normal class into an enum::
+
+      >>> from enum import Enum, simple_enum
+      >>> @simple_enum(Enum):
```

Comment:
if we were keeping the docs... extra trailing : and I believe auto() needs to be imported below?